docs/docs/gatsby-cli.md

```diff
@@ -66,121 +66,121 @@ gatsby new
    (Use a different starter)
 ```
 
-See the [Gatsby starters docs](https://www.gatsbyjs.org/docs/gatsby-starters/) for more details.
+अधिक विवरण के लिए [Gatsby शुरुआत डॉक्स](https://www.gatsbyjs.org/docs/gatsby-starters/) देखें।
 
-### `develop`
+### `विकसित करना`
 
-Once you've installed a Gatsby site, go to the root directory of your project and start the development server:
+एक बार जब आप एक Gatsby साइट इनस्टॉल कर लेते हैं, तो अपनी प्रॉजेक्ट की रूट फोल्डर मे जाएँ और डेवेलपमेंट सर्वर शुरू करें:
 
 `gatsby develop`
 
-#### Options
+#### विकल्प
 
-|     Option      | Description                                     |
+|     विकल्प      | विवरण                                     |
 | :-------------: | ----------------------------------------------- |
-| `-H`, `--host`  | Set host. Defaults to localhost                 |
-| `-p`, `--port`  | Set port. Defaults to 8000                      |
-| `-o`, `--open`  | Open the site in your (default) browser for you |
-| `-S`, `--https` | Use HTTPS                                       |
+ `-H`, `--host`  | होस्ट सेट करें। डिफॉल्ट लोकलहोस्ट             |
+| `-p`, `--port`  | पोर्ट सेट करें। डिफॉल्ट ८००० |                      |
```

Comment:
```
| `-p`, `--port`  | पोर्ट सेट करें। डिफॉल्ट ८००० |                      |
```
=>
```
| `-p`, `--port`  | पोर्ट सेट करें। इसकी डिफ़ॉल्ट वैल्यू 8000 है         |
```

Reply:
You missed this

Reply:
again outdated code.

Reply:
Please check the above snapshot or this url 

https://github.com/gatsbyjs/gatsby-hi/pull/11/files#diff-811f4ccc1e57432512ee141d439fe389R82

You have two `|` at the end